src/lib/server.ts

```diff
@@ -291,9 +313,16 @@ export class ZwavejsServer extends EventEmitter {
     if (!this.driver.ready) {
       throw new Error("Cannot start server when driver not ready");
     }
+
+    // Create log transport for server and update the drivers log configuration
+    this.serverTransport = new EventEmitterLogTransport(this);
+    const transports = this.driver.getLogConfig().transports;
+    transports.push(this.serverTransport);
+    this.driver.updateLogConfig({ transports });
```

Comment:
Is it possible that this is called multiple times? I guess not, but if it is, you should probably check if the transport was already registered.

Reply:
It shouldn't be possible the way the setup is done: https://github.com/zwave-js/zwave-js-server/blob/master/src/bin/server.ts#L69-L77